src/components/PaginationFooter/context/PaginationProvider.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { createContext, useContext, useMemo } from 'react';
+import type { stardust } from '@nebula.js/stardust';
+import type { FooterStyle, PaginationContentProps } from '../types';
+import getFooterStyle from '../utils';
+import useTranslations from '../../../hooks/useTranslations';
+
+interface IPaginationProvider extends PaginationContentProps {
+  footerStyle: FooterStyle;
+  tabIndex: number;
+  width: number;
+}
+
+interface PaginationProviderProps extends PaginationContentProps {
+  children: React.ReactNode;
+}
+
+export const areTabStopsEnabled = (keyboard: stardust.Keyboard) => !keyboard.enabled || keyboard.active;
```

Comment:
This is used quite frequently in sn-table. I'd say if we want this in one place, it should be on higher level utils file. at the same time, it is tiny and not the end of the world if it is defined in multiple places...

Reply:
I won't do that in this PR at least